test/source/TestAlgorithm.cpp

```diff
@@ -116,6 +118,57 @@ struct TestObjectNegate : public eastl::unary_function<TestObject, TestObject>
 		{ return TestObject(-a.mX); }
 };
 
+struct MissingMoveConstructor
```

Comment:
Look at "ConceptImpls.h" these testing types are already added there and if one is missing it should be added there so its can be accessed via all tests.